private/protocol/restxml/build_bench_test.go

```diff
@@ -14,7 +14,11 @@ import (
 	"github.com/aws/aws-sdk-go-v2/internal/awstesting/unit"
 	"github.com/aws/aws-sdk-go-v2/private/protocol/restxml"
 	"github.com/aws/aws-sdk-go-v2/service/cloudfront"
+	"github.com/aws/aws-sdk-go-v2/service/cloudfront/enums"
+	"github.com/aws/aws-sdk-go-v2/service/cloudfront/types"
```

Comment:
Nit suggest usign `cfEnums` and `cfTypes`. and us camel casing instead of underscores.